user-console/src/main/resources/org/pentaho/mantle/public/browser/index.jsp

```diff
@@ -286,9 +286,11 @@
     }
   }
 
+  // With the fix BACKLOG-23730, server-side and client-side code uses centralized logic to check if user
+  // can download/upload content
   function checkDownload() {
     $.ajax({
-      url: CONTEXT_PATH + "api/authorization/action/isauthorized?authAction=org.pentaho.security.administerSecurity",
+      url: CONTEXT_PATH + "api/repo/files/canDownload?dirPath=",
```

Comment:
If we're in a context where we do not have a selected `dirPath` yet, can we keep the endpoint clearer, by not including the queryParam?

Reply:
You're right, I'll fix this!